monai/networks/blocks/fcn.py

```diff
@@ -20,7 +20,7 @@
 from monai.networks.layers.factories import Act, Conv, Norm
 from monai.utils import optional_import
 
-models, _ = optional_import("torchvision", "0.5.0", name="models")
+models, _ = optional_import("torchvision", name="models")
```

Comment:
@yiheng-wang-nv I've changed `torchvision==0.5.0` to `torchvision`, because torchvision 0.5.0 is with torch1.4. do you see any issue here? it seems to work fine with torch1.6+torchvision0.7.0